docs/_webhooks.md

```diff
@@ -0,0 +1,146 @@
+# Webhooks (Coming Soon!)
+
+Webhooks allow events to be sent to a specified url when an event matches the provided topic.
+
+## Topic
+
+A topic is composed of a subject, the subject's id, a verb, and an output type. For example the topic "player.examplePlayer.played.match" would receive events when a player with the id of examplePlayer played a match and the payload would be of type "match".
+
+### Supported Topics
+
+- player.playerId.played.match
+- webhook.webhookId.verify.event
+
+## Post a Webhook
+
+```shell
+curl -XPOST "https://api.dc01.gamelockerapp.com/shards/na/webhooks" \
+  -H "Authorization: Bearer <api-key>" \
+  -H "Accept: application/vnd.api+json"
+```
+
+This endpoint creates a webhook
+
+Note: A verification event will be sent to the url (see supported events above). If the request does not succeed, the webhook will not be created.
```

Comment:
Here we refer to 'supported events', where as previously we referred to 'supported topics'. Should we keep terminology the same?